features/provider/finance/invoices.feature

```diff
@@ -0,0 +1,43 @@
+@ignore-backend @javascript
+Feature: Show all provider's invoices
+  In order to manage invoices
+  As a provider
+  I want to be able to see all invoices issued by me
+
+  Scenario: Display years of invoices when there are invoices
+    Given a provider is logged in on 1st January 2011
+    And a buyer "zoidberg" signed up to provider "foo.3scale.localhost"
```

Comment:
```suggestion
    And a buyer signed up to the provider
```

Reply:
There is an inconvenience in the `a buyer signed up to the provider`, and it's that it creates an application (which we don't need here), and actually this step fails with a:
```
Provider has no default application plan
```

We would need to add a default application plan to make it work, but again - we don't need an application in this scenario.

Also, for the last (and the most important) scenario `Display only years belonging to the current provider` it is better to have the buyer's names explicit, because it may be confusing, as the different buyers belong to different providers.